app/routes/events/components/EventDetail/index.js

```diff
@@ -295,26 +278,50 @@ export default class EventDetail extends Component<Props> {
               </Flex>
             </ContentMain>
             <ContentSidebar>
-              <InfoList items={infoItems} />
+              <div>
+                <Icon name="time-outline" style={{ marginRight: '6px' }} />
+                <strong>
+                  <FromToTime from={event.startTime} to={event.endTime} />
+                </strong>
+              </div>
+              <div>
+                <Icon name="pin-outline" style={{ margin: '0 3px' }} />
+                <strong>{' ' + event.location}</strong>
+              </div>
+              {event.mazemapPoi && (
+                <div>
+                  <a
+                    onClick={() =>
+                      this.setState({ mapIsOpen: !this.state.mapIsOpen })
+                    }
+                  >
+                    {' '}
+                    Vis kart
+                  </a>
+                  {this.state.mapIsOpen && (
+                    <MazemapEmbed mazemapPoi={event.mazemapPoi} />
+                  )}
+                </div>
+              )}
               {event.isPriced && (
                 <div className={styles.paymentInfo}>
-                  <strong>Dette er et betalt arrangement</strong>
-                  <InfoList items={paidItems} />
+                  <InfoList
+                    items={[
+                      { key: '$', value: `${event.priceMember / 100},-` },
+                    ]}
+                  />
                 </div>
               )}
-              {event.mazemapPoi && (
-                <MazemapEmbed mazemapPoi={event.mazemapPoi} />
-              )}
               {['OPEN', 'TBA'].includes(event.eventStatusType) ? (
                 <JoinEventForm event={event} />
               ) : (
                 <Flex column>
-                  <h3>Påmeldte</h3>
+                  <h3 style={{ marginTop: 12 }}>Påmeldte</h3>
                   {registrations ? (
                     <Fragment>
                       <UserGrid
-                        minRows={2}
-                        maxRows={2}
+                        minRows={1}
+                        maxRows={1}
```

Comment:
I am really uncertain about this 👀

Reply:
Kindof like having two rows as seeing familiar faces in the preview could boost registrations

Reply:
I like two rows. I think you should do something about the size properties though, as it's now a fixed size, making it an ugly empty box whenever no one is registered.

Reply:
ooh, this went unnoticed :grimacing: